### ISSUE TITLE
overall mod idea for fixing the render break for custom models (unsure if possable)

### ISSUE THREAD

**Issue by SteamDoge9000** [Yesssssman/epicfightmod#871]
ik that this mod has some issues with custom armors and not all devs are willing to go though the process to initgreate every armor for it to work, so what if this was done automatically by the mod itself on its first startup/whenever a new mod with armor that's added is detected

what I'm suggesting is something that I'm not sure if its possible, but MIGHT be a overall fix to this damning issue.

what if the mod itself was able to pull all textures that are registered as armor for the player and then takes its textures and reworks the texture model in a way that would fix the bugged look while at the same time overriding the original view of the (buged) texture (in a way that's like how cosmetic slots overides armor textures with a preferred visual)? 

or

what if a command was added that when you hold an armor that's model is bugging with EFM custom model that when ran when held would pull/dump its texture and model stuff and then integrate it in a way that fixes it, this data would NOT alter the mod but be stored instead in the config file where if its on a specific statement in the config (that gets added to it when you run this command) will overide the original texture when the custom model is being used (confined based if the model reverse back to normal in miner mode) with the custom generated one that was made compatible with EFM model

idk if this is possible, but if it is and could be done, this would, in my mind, theoretically fix this damning issue once and for all. And the best part of it is, that once its stored in the config, it can be distributed in the same way that modpacks have custom configs. 

TLDR: a system where armors textures breaking with the EFM will be "dumped" and then adjusted to work with the custom player model when a command is set and is stored within this config file; this will be pulled and override the original texture (or only when in the battle mode if the limit model config is set to true) in a way like amor cosmetic mods and then instead render the mod generated amor texture made compatible. this would be only used if its in a list, a command can be run to add the given item your holding to a list, almost like how you can auto-detect weapons in the EFM config. this would only generate once if: No following files already exist in the mod file, AND if the item is on the custom list (like with auto-switching per see)

**Comment by Yesssssman** [comment]
It's not just the problem of textures, it's the problem of the rendering method as well. If the custom armor is rendered in the way Epic Fight didn't support, it never can be fixed. For instance, Ice and Fire custom armors are rendered normally, but the Dungeons Gears custom armors are broken because they defined the model instance on its class and didn't store it in a vanilla variable. So the custom model generator in Epic Fight doesn't work well.
